src/terraform/structure/terraform_parser_test.go

```diff
@@ -50,19 +50,31 @@ func TestTerraformParser_SkipResourceByComment(t *testing.T) {
 		assert.Empty(t, p.GetSkipResourcesByComment())
 	})
 
-	t.Run("One resource with skip comment, only that resource added to skipResourcesByComment slice", func(t *testing.T) {
-		// Initialize TerraformParser and parse file with one resource containing skip tag
-		p := &TerraformParser{}
-		p.Init("../../../tests/terraform/skipComment/", nil)
-		defer p.Close()
-		filePath := "../../../tests/terraform/skipComment/skipOne.tf"
-		_, err := p.ParseFile(filePath)
-		if err != nil {
-			t.Errorf("failed to read hcl file because %s", err)
+}
+
+func TestParseTagAttribute(t *testing.T) {
+	filePath := "../../../tests/terraform/forLoop/main.tf"
+	expectedTags := map[string]string{
+		"c": "d",
+	}
+	parser := &TerraformParser{}
+	parsedBlocks, err := parser.ParseFile(filePath)
+	if err != nil {
+		t.Fatalf("Failed to parse file: %v", err)
+	}
+	for _, block := range parsedBlocks {
+		hclBlock := block.GetRawBlock().(*hclwrite.Block)
+		tagsAttributeName, _ := parser.getTagsAttributeName(hclBlock)
+		tagsAttribute := hclBlock.Body().GetAttribute(tagsAttributeName)
+		if tagsAttribute != nil {
+			tagsTokens := tagsAttribute.Expr().BuildTokens(hclwrite.Tokens{})
+			parsedTags, _ := parser.parseTagAttribute(tagsTokens)
+			if block.GetResourceName() == "bucket_var_tags" {
```

Comment:
Can we test more than just this resource?

Reply:
@ChanochShayner 
Yes we can test also another resource 
but we need to compare it with another expected result